raiden/utils/filters.py

```diff
@@ -146,6 +143,7 @@ def get_new_entries(self, target_block_number: BlockNumber) -> List[Dict[str, An
             return result
 
     def get_all_entries(self, block_number: BlockNumber = None):
+        # TODO Is this method used anywhere?
```

Comment:
Please create an issue for this.

Reply:
I removed the comment and will open an issue instead.